<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="DESKTOP-QB5O9P3" failures="2" tests="2" name="Test1(failed)" time="0.007" errors="0" timestamp="2022-07-27T14:36:22 PDT">
  <testcase classname="howToRunFailedTCOnlyUsingTestNG.NewTestNGClass3" name="n" time="0.001">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at howToRunFailedTCOnlyUsingTestNG.NewTestNGClass3.n(NewTestNGClass3.java:11)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- n -->
  <testcase classname="howToRunFailedTCOnlyUsingTestNG.NewTestNGClass3" name="p" time="0.001">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at howToRunFailedTCOnlyUsingTestNG.NewTestNGClass3.p(NewTestNGClass3.java:22)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- p -->
  <testcase classname="howToRunFailedTCOnlyUsingTestNG.NewTestNGClass3" name="o" time="0.0">
    <ignored/>
  </testcase> <!-- o -->
  <testcase classname="howToRunFailedTCOnlyUsingTestNG.NewTestNGClass3" name="q" time="0.0">
    <ignored/>
  </testcase> <!-- q -->
</testsuite> <!-- Test1(failed) -->
